### PR TITLE
Update peach mutator to python3

### PR DESCRIPTION
**Pull request by mpherman2**

### CODE REVIEW

File: docker/base/requirements.txt [hunk 1]
```diff
@@ -1,8 +1,11 @@
 crcmod==1.7
 future==0.17.1
+lxml==4.5.0
```
Comment (Dor1s) [inline docker/base/requirements.txt:3]: I'm not sure if this is needed after adding the same deps to `src/requirements.txt`

Comment (inferno-chromium) [reply docker/base/requirements.txt:3]: https://github.com/google/clusterfuzz/pull/1712

Comment (Dor1s) [reply docker/base/requirements.txt:3]: awesome!

Comment (oliverchang) [reply docker/base/requirements.txt:3]: These deps here are also deprecated and Python 2 only.

**Pull request merged**